alvr/server_openvr/cpp/platform/win32/OvrDirectModeComponent.h

```diff
@@ -71,5 +78,8 @@ class OvrDirectModeComponent : public vr::IVRDriverDirectModeComponent {
     uint64_t m_targetTimestampNs;
     uint64_t m_prevTargetTimestampNs;
 
+    vr::HmdRect2_t m_viewProj[2];
+    vr::HmdMatrix34_t m_eyeToHead[2];
```

Comment:
I'm not convinced about saving the view parameters in the DirectMode class. I think they should stay inside `m_pD3DRender`